### PR TITLE
New page added with Quotes section

### PR DESCRIPTION
**Pull request by jpnaveenkumar**
Following issues are fixed 👍 
#7 integrated Quotes api to the app ( displays new Quotes on every page refresh)
#3 added new page to the app

### CODE REVIEW

File: src/components/Quotes.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    constructor(props)
+    {
+        super(props);
+        this.state = {
+            isLoading : true,
+            quote : "fetching Quotes ..."
+        }
+    }
+    getQuote = () =>{
+        fetch("https://ron-swanson-quotes.herokuapp.com/v2/quotes/1")
```
Comment (aligg) [inline src/components/Quotes.js]: lol ron swanson quotes!?

File: src/components/Quotes.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+                    </div>
+                    <div>
+                        {content}
+                    </div>
+                </div>
+            </div>
+        );
+    }
+}
+export default Quotes;
```
Comment (aligg) [inline src/components/Quotes.js]: missing new line at the end of the file I think!

**Review by aligg — APPROVED** [review]

**Comment by aligg** [comment]
Going ahead and merging this! This makes me think that something useful to add to this repo would also be a linter so we can build a shared style for the repo. Thanks for your contributions!

File: src/components/Quotes.js (hunk 1 above)
Comment (jpnaveenkumar) [reply src/components/Quotes.js:15]: yeah 😸

**Pull request merged**